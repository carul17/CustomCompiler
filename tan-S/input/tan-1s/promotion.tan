main {
	var a := 'a' + 5.5;
	var  b := 'b' + 5;
	print b \n;
    print a \n;
}