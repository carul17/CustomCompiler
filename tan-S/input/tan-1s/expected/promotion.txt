103
102.500000
